test/parallel/test-repl-eval.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const repl = require('repl');
+
+{
+  let evalCalledWithExpectedArgs = false;
+
+  const options = {
+    eval: common.mustCall((cmd, context) => {
+      // Assertions here will not cause the test to exit with an error code
+      // so set a boolean that is checked in process.on('exit',...) instead.
+      evalCalledWithExpectedArgs = (cmd === 'foo\n' && context.foo === 'bar');
```

Comment:
That's right. They're just there for clarity. If their objectionable, I can remove them.